src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const basicCost = dailyCost * days;
+  const basicDiscount = 20;
+  const higherDiscount = 50;
+
+  if (days >= 3 && days < 7) {
```

Comment:
Maybe we can reorganize checking the conditions to avoid using `&&`? :)
What if we first checked for large discount and then for smaller one?